### PR TITLE
Update MrCoin exchange links

### PR DESCRIPTION
**Pull request by ejulcsi**

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Building with commit 8844592920338dcb969bac2fd5f2e7a9a922ae05

https://app.netlify.com/sites/augmint/deploys/5cffca1f4e08180008c763a1

File: src/components/BaseComponents.js [hunk 1]
```diff
@@ -80,7 +80,11 @@ export const Validations = {
     },
 
     minMrCoinTokenAmount: value => {
-        return value < 3 ? `Amount must be at least: 3 A-EUR` : undefined;
+        return value < 25 ? `Amount must be at least: 25 A-EUR` : undefined;
+    },
+
+    minMrCoinEurAmount: value => {
+        return value < 15 ? `Amount must be at least: 15 EUR` : undefined;
```
[the remark this answers is not in the document]
Comment (phraktle) [reply src/components/BaseComponents.js]: Also, `BaseComponent` should only have generic validations. Validations specific to a component should be encapsulated in the component.

File: src/components/BaseComponents.js [hunk 2]
```diff
@@ -79,12 +79,8 @@ export const Validations = {
         }
     },
 
-    minMrCoinEurAmount: value => {
-        return value < 25 ? `Amount must be at least: 25 EUR` : undefined;
-    },
-
-    minMrCoinAEurAmount: value => {
-        return value < 15 ? `Amount must be at least: 15 A-EUR` : undefined;
+    minMrCoinAmount: (minvalue, currency) => value => {
```
Comment (phraktle) [inline src/components/BaseComponents.js]: should be just `minAmount`

File: src/containers/funding/components/AddWithdrawForm.js [hunk 3]
```diff
@@ -18,7 +18,12 @@ export const ADDFUND = "addFunds";
 class AddWithdrawForm extends React.Component {
     constructor(props) {
         super(props);
-        this.state = { orderDirection: ADDFUND, amount: "" };
+        this.state = {
+            orderDirection: ADDFUND,
+            amount: "",
+            minEUR: Validations.minMrCoinAmount(FUNDS[0].eurLimit, "EUR"),
```
Comment (phraktle) [inline src/containers/funding/components/AddWithdrawForm.js]: should not be in state (use a `const` or move it to `funds.js`)

**Pull request merged**